include/boost/astronomy/coordinate/arithmetic.hpp

```diff
@@ -120,6 +122,401 @@ auto cross
         >(result);
 }
 
+//!Returns the cross product of representation1(cartesian differential) and representation2
```

Comment:
Change representation1 -> differential1.
Same for representation2
Rest all looks good to me